crates/opencascade-sys/build.rs

```diff
@@ -36,6 +37,8 @@ fn main() {
     cxx_build::bridge("src/lib.rs")
         .cpp(true)
         .flag_if_supported("-std=c++11")
+        .define("_USE_MATH_DEFINES", "TRUE")
+        .include(format!("{}", dst.join("inc").display()))
```

Comment:
Is CMake generating this directory only for windows targets?

Reply:
Seems to be. The structure is different in subtle ways. 

One option is to add conditionals to enable/disable these different paths. I opted not to do that yet. Once a pattern emerges it might be a good idea. 

Im not sure what OCCT does for `macos` or `webassembly`.

Reply:
Ah sorry, I should mention that I develop mainly on MacOS so that target should be well tested.

I'm currently not planning to target web assembly, though I may have plans in the future to have a model viewer that accepts web assembly blobs which generate instructions for generating an opencascade model.

Reply:
Fair enough. I am primarily on Linux myself so that covers those two. I will probably need webassembly at some point but I am in no rush for it. 

I'm working on a project similar to OpenSCAD. Last year I did the webassembly port of OpenSCAD and judging from the community reaction, I will probably need it for this project as well.

Reply:
Nice, sounds like the two ideas we have in mind might be similar.

Happy to accept PRs for adding web assembly support of course, assuming it doesn't add any huge restrictions on the codebase. I just don't need it at the moment and won't be adding it myself in the immediate future.

Reply:
Ill send them your way for sure. 

My project is still super early days but its usable in basic ways.
https://github.com/DSchroer/model-script
There is some example code in the "examples" dir.

Reply:
I wouldnt have gotten so far if this crate didnt exist. I really appreciate the work you did here.

Reply:
Really cool to get a usable, real-world object as a result of some Rust hacking :)

> I did get a little 3D editor in it so it really feels like working with OpenSCAD at this point.

MacOS has a native STL viewer that I've been using as a crutch, otherwise I probably would have gotten frustrated and made a viewer as well.

> I wouldnt have gotten so far if this crate didnt exist. I really appreciate the work you did here.

Thank the opencascade devs! And dtolnay for the `cxx` crate :) I'm just gluing things together here, though I did suffer through the initial work of getting `build.rs` working properly, I'll take credit for that haha

Reply:
I'll clone it and give it a spin!